Documentation/network/servicemesh/gateway-api/host-network-mode.rst

```diff
@@ -0,0 +1,70 @@
+Host network mode
+#################
+.. note::
+    Supported since Cilium 1.16+
+
+This mode allows you to expose the Cilium GatewayAPI Gateway
+directly on the host network.
+This is useful in some edge cases where one doesn't want, or can't, use
+LoadBalancer type services (dev environments, cluster-external
+loadbalancer, ...).
+
+The host network mode can be enabled via Helm flag
+``gatewayAPI.hostNetwork.enabled=true``.
+Once enabled, the host network ports can be specified on the ``Gateway``
+resources via ``spec.listeners.port``. The port must be unique per ``Gateway``
+resource and higher than ``1023``.
+
+.. warning::
```

Comment:
I think this is more of a `note` than a `warning` - except the following

> Be aware that misconfiguration might result in port clashes. Configure unique ports that are still available.

Maybe worth to change into two blocks (note and warning)?